src/app/pages/home/home.component.html

```diff
@@ -1 +1,11 @@
-<h1>Home</h1>
+<div>
+    <h1>Movies</h1>
+    <div class="movies">
+        <app-movie class="movie"
+                   *ngFor="let item of movies"
+                   [movie]="item"
+        ></app-movie>
+    </div>
+
```

Comment:
Please, remove all unnecessary strings in templates and components

Reply:
done